tocrify/api/mets.py

```diff
@@ -37,7 +37,7 @@ def __init__(self, node):
         The constructor.
         """
         self.phys_id = node.get("ID")
-        self.hocr_id = node.xpath("./mets:fptr[starts-with(@FILEID, \"HOCR\")]", namespaces=ns)[0].get("FILEID")
+        self.hocr_id = node.xpath("./mets:fptr[contains(@FILEID, \"HOCR\")]", namespaces=ns)[0].get("FILEID")
```

Comment:
Maybe some samples of valid IDs in a comment so users with different ID schema can reuse?